daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Compiler.scala

```diff
@@ -487,8 +494,66 @@ private[lf] final class Compiler(
 
   private[this] def translateChoiceBody(
       env: Env,
-      typeId: TypeConName,
+      tmplId: TypeConName,
```

Comment:
Why do we change that ?  We use `typeId` here to underline it may be an interface. 
Anyway if you want to change prod code, please create a separate PR.

Reply:
Actually no, `translateChoiceBody` is only used for templates, so renaming the variable to `tmplId` is better for consistency.

Reply:
fair enough